tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es6",
+    "target": "ESNext",
```

Comment:
Why do we need to change that? 

https://www.typescriptlang.org/tsconfig#target

> The special ESNext value refers to the highest version your version of TypeScript supports. This setting should be used with caution, since it doesn’t mean the same thing between different TypeScript versions and can make upgrades less predictable.

Reply:
@reneaaron thanks for catching this. I think this is another change related to getting signPsbt working. Reverted.